### ISSUE TITLE
refmt errors are visible only in logs

### ISSUE THREAD

**Issue by Coobaha** [reasonml-editor/reasonml-idea-plugin#64]
plugin version: 
0.51.1
### Description
It was visible before but with recent versions it fails silently. 

Currently the only way to see error is to check IDE log files.

Code to reproduce: `let test = "string" ++ "\ufff0";`

```
2018-05-21 09:20:17,238 [ 169413]   WARN -                 ReasonML.refmt - File "", line 2, characters 24-26:
Warning 14: illegal backslash escape in string.
```

**Comment by Coobaha** [comment]
Right, it is visible in bsb output but hard to find, my bad ;)  but some IDE feedback would be great to have in this case.

**Comment by giraud** [comment]
should be fixed in 0.52
